### PR TITLE
BaSP-TG-52: React Routes installed and added

### PR DESCRIPTION
**Pull request by Eju97**
-Installed React Routes.
-Redone Layout routes.

### CODE REVIEW

File: src/Components/Admins/index.js [hunk 1]
```diff
@@ -69,9 +70,10 @@ const Admins = () => {
             );
           })}
           <tfoot>
-            <button type="button" onClick={() => window.location.assign(`/admins/form?`)}>
-              Create
-            </button>
+            <Link to="/admins/form">Create</Link>
+            {/* <button type="button" onClick={() => window.location.assign(`/admins/form?`)}>
```
Comment (GinaSchP) [inline src/Components/Admins/index.js:74]: please remove this comment

**Review by GinaSchP — DISMISSED** [review]

File: src/Components/Admins/index.js (hunk 1 above)
Comment (Eju97) [reply src/Components/Admins/index.js]: Done!

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -69,9 +71,9 @@ const Admins = () => {
             );
           })}
           <tfoot>
-            <button type="button" onClick={() => window.location.assign(`/admins/form?`)}>
-              Create
-            </button>
+            <Link to="/admins/form">
+              <button>Create</button>
```
Comment (lmalvasia) [inline src/Components/Admins/index.js:75]: I think you should do:
```
<button type="button" onClick={() => history.push.....)}>
              Create
            </button>
```

File: src/Components/Admins/Form/index.js [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import styles from './form.module.css';
 import { useState, useEffect } from 'react';
 import Modal from '../Modal/modal';
+import { Link, useParams } from 'react-router-dom';
 
 const Form = () => {
-  const paramsURL = new URLSearchParams(window.location.search);
-  const adminId = paramsURL.get('id');
+  const params = useParams();
```
Comment (lmalvasia) [inline src/Components/Admins/Form/index.js:7]: Well done 👍

File: src/Components/SuperAdminsForm/index.js [hunk 4]
```diff
@@ -118,6 +120,9 @@ function SuperAdminsForm() {
           <button onClick={onSubmit} type="button">
             Apply
           </button>
+          <Link to="/super-admins">
+            <button>Go Back</button>
```
Comment (lmalvasia) [inline src/Components/SuperAdminsForm/index.js:124]: If you are going to use a button, you should the onClick function with the `goBack()` method from history.

Comment (lmalvasia) [reply src/Components/SuperAdminsForm/index.js:124]: And remove the Link

File: src/Components/TimeSheets/Form/timesheetForm.js [hunk 5]
```diff
@@ -203,7 +203,7 @@ const Form = () => {
           </div>
         </div>
         <button type="submit">Add</button>
-        <a href="'/time-sheets'">Go Back</a>
+        <Link to="/time-sheets">Go Back</Link>
```
Comment (lmalvasia) [inline src/Components/TimeSheets/Form/timesheetForm.js:206]: I recommend you to use the button here and the goBack function from history.

**Review by lmalvasia — APPROVED** [review]

**Pull request merged**